app/src/main/java/com/hfad/beeradviser/BeerExpert.java

```diff
@@ -8,7 +8,7 @@
     //TODO: Complete the Model with private members, a constructor, and getter/setter methods
 
 
-    String getRecommendation(String color) {
+    static String getRecommendation(String color) {
```

Comment:
Instead of making this static, refactor the method to use a new "mColor" instance variable